### ISSUE TITLE
What is the structure of train dataset ?

### ISSUE THREAD

**Issue by OberstWB** [ronghuaiyang/arcface-pytorch#22]
What is the structure of train dataset ? Could you please show me? Thx!

**Comment by OberstWB** [comment]
It seems that all train images are in one folder, what is the way you name per image?

**Comment by changhongjian** [comment]
I also want to know what's the training set of the pretrained model, is that VGGface2?

**Comment by zza584231732** [comment]
> What is the structure of train dataset ? Could you please show me? Thx!

I also want to know, can you tell me? Thank you

**Comment by 1LOVESJohnny** [comment]
Hi, do you figure out the structure of training data? I want to know how to generate the training data list. Can anybody give me a clue?

**Comment by luluvi** [comment]
> Hi, do you figure out the structure of training data? I want to know how to generate the training data list. Can anybody give me a clue?

How do you get the data set training, can you give me some ideas?
thank you!

**Comment by ronghuaiyang** [comment]
the data list structure is:

xxx.jpg  label

the label start from 0
